modules/installation-configuration-parameters.adoc

```diff
@@ -358,6 +366,12 @@ If you deploy to a custom subnet, you cannot specify an external DNS server to t
 
 |A UUID as a string. For example, `fa806b2f-ac49-4bce-b9db-124bc64209bf`.
 
+|`platform.openstack.zones`
```

Comment:
We do not support this parameter and do not plan to do it yet.

Reply:
This is indeed causing a bit of confusion as seen in https://bugzilla.redhat.com/show_bug.cgi?id=1880000.
We're only allowing setting the zones via the `compute.platform.openstack.zones` and `controlPlane.platform.openstack.zones` parameters, consistently with how other platforms do it.

Reply:
I have amended the CEE enablement slides on which I had mistakenly mentioned this non-existent platform-level property. My bad, sorry for the confusion.

Reply:
Will strike it. Thanks!